src/net.cpp

```diff
@@ -2489,6 +2500,7 @@ void NetCleanup()
 
 void RelayTransaction(const CTransactionRef &ptx, const bool fRespend)
 {
+    // TODO call a GetSize() function that caches the tx size
```

Comment:
:+1:

Reply:
I have left a few optimization notes to ourselves in the code.